Service/Microsoft.Health.DeIdentification.Web/Controllers/FhirController.cs

```diff
@@ -0,0 +1,76 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Serialization;
+using Hl7.FhirPath;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyModel;
+using Microsoft.Health.DeIdentification.Contract;
+using Microsoft.Health.DeIdentification.Fhir;
+using Microsoft.Health.DeIdentification.Local;
+using Microsoft.Health.DeIdentification.Web.Models;
+using Microsoft.Health.Fhir.Anonymizer.Core;
+using Newtonsoft.Json;
+using System;
```

Comment:
nit, we usually order namespaces by System.* first, and others by alphabetic order

Reply:
Ok, I will modify order later